_unittest/test_98_Icepak.py

```diff
@@ -53,8 +53,11 @@ def setup_class(self):
 
     def teardown_class(self):
         self.aedtapp._desktop.ClearMessages("", "", 3)
-        self.aedtapp.close_project(src_project_name, False)
-        self.aedtapp.close_project(self.aedtapp.project_name)
+        try:
+            self.aedtapp.close_project(src_project_name, False)
+            self.aedtapp.close_project(self.aedtapp.project_name)
+        except:
+            pass
```

Comment:
@maxcapodi78 Shouldn't we raise an exception in the `except` block?

Reply:
Hi @MaxJPRey that's in unit test. I do it to reduce random IPY errors in UT

Reply:
ok. Make sense.